packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -210,6 +213,20 @@ function isSideloadingSupportedForWebHost(app: OfficeApp): boolean {
   return false;
 }
 
+async function getOutlookVersion(): Promise<string | undefined> {
+  try {
+    const OutlookInstallPathVersionRegistryKey = `HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Office\\ClickToRun\\Configuration`;
+    const key = new registry.RegistryKey(`${OutlookInstallPathVersionRegistryKey}`);
```

Comment:
There's no need for the OutlookInstallPathVersionRegistryKey variable.  Just put it directly:

`... new RegistryKey("HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Office\\ClickToRun\\Configuration");`